tests/test_pipe_viz.py

```diff
@@ -55,29 +55,31 @@ def viz_pipeline_fixture(config, filter_probs_df):
     return pipe
 
 
-def test_call_before_run(viz_pipeline: Pipeline, tmp_path: str):
+@pytest.mark.parametrize("run_before,run_after", [
+    (True, False),
+    (False, False),
+    (False, True),
+])
+def test_png(viz_pipeline: Pipeline, tmp_path: str, run_before: bool, run_after: bool):
```

Comment:
Because `(True, True)` isn't supported.
I wanted to test that we could generate the viz graph in all of the following situations:
1) Without executing the pipeline
2) before running the pipeline
3) after running the pipeline.